packages/common/src/MultiVersionTestHelpers.js

```diff
@@ -13,10 +13,9 @@ const SUPPORTED_CONTRACT_VERSIONS = [
   { contractType: "Perpetual", contractVersion: "2.0.1" },
```

Comment:
Looks like you can also removes versions from here?

Reply:
well, the way this PR is set up is that the bot's still _support_ these versions (all the branching logic is still in them ) but they don't have tests for them anymore. In a follow-up PR we can go into the bot implementation and remove the branching logic. I thought it best to separate this into a two-step process to minimize code in this PR (already a large diff).